gradle.properties

```diff
@@ -79,7 +77,7 @@ org.gradle.configureondemand = false
 org.gradle.daemon = true
 org.gradle.jvmargs = -Xmx3g
 org.gradle.parallel = true
-#org.gradle.workers.max = 3
+org.gradle.workers.max = 8
```

Comment:
Why are we setting max workers, instead of letting Gradle check the CPU count on the host?

Reply:
That's a mistake. I thought were were previously setting this to 8 here. I don't know why I thought that. I'll fix it.